app/src/main/java/ch/sdp/vibester/activity/GameEndingActivity.kt

```diff
@@ -40,6 +40,15 @@ class GameEndingActivity : AppCompatActivity() {
 
         setContentView(R.layout.activity_game_ending_screen)
 
+        if (intent.hasExtra("Winner Name")) {
+            val winner = intent.getStringExtra("Winner Name")
+            /*
+            if (winner!=null) {
```

Comment:
I would suggest either removing the commented code or adding a fixme explanation